src/HypLSP8.sol

```diff
@@ -0,0 +1,74 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity >=0.8.19;
+
+import { TokenRouter } from "@hyperlane-xyz/core/contracts/token/libs/TokenRouter.sol";
+
+import { LSP8IdentifiableDigitalAssetInitAbstract } from
+    "@lukso/lsp8-contracts/contracts/LSP8IdentifiableDigitalAssetInitAbstract.sol";
+
+import { _LSP4_TOKEN_TYPE_TOKEN } from "@lukso/lsp4-contracts/contracts/LSP4Constants.sol";
+
+import { _LSP8_TOKENID_FORMAT_NUMBER } from "@lukso/lsp8-contracts/contracts/LSP8Constants.sol";
+
+/**
+ * @title Hyperlane LSP8 Token Router that extends LSP8 with remote transfer functionality.
```

Comment:
Suggest also adding a link to the HypERC721 for reference

```suggestion
 * @title LSP8 version of the Hyperlane ERC721 Token Router
```

Reply:
Like below

<img width="1013" alt="image" src="https://github.com/user-attachments/assets/1210f690-6a24-46e7-a082-ce3ee7026d3d">